stack/docker-compose.yml

```diff
@@ -34,6 +38,19 @@ services:
       - internal
       - keycloak
 
+  postgres-backend:
+    image: postgres:13.2
+    container_name: postgres-backend
+    environment:
+      POSTGRES_DB: praktikumsplaner
+      POSTGRES_USER: db-backend-user
+      POSTGRES_PASSWORD: db-backend-password
```

Comment:
User and Password in plain text?

Reply:
this is the user and password for the local running infrastructure. When running the app for real operations must take care for their credentials. These file should not be used to run the software in production.